plugin-locationcomponent/src/main/java/com/mapbox/maps/plugin/locationcomponent/DefaultLocationProvider.kt

```diff
@@ -191,8 +190,8 @@ class DefaultLocationProvider @VisibleForTesting(otherwise = PRIVATE) internal c
     private val locationProviderWeakReference: WeakReference<DefaultLocationProvider> =
       WeakReference(locationProvider)
 
-    override fun onSuccess(result: LocationEngineResult) {
-      result.lastLocation?.let { location ->
+    override fun onSuccess(result: LocationEngineResult?) {
+      result!!.lastLocation?.let { location ->
```

Comment:
Can null arrive here?

Reply:
This is outdated, common SDK introduced this breaking change and has reverted it back.